lib/morse_translator.rb

```diff
@@ -40,4 +40,14 @@ def initialize
                     " " => " "}
   end
 
+  def turns_into_chars(word)
+    word.chars
+  end
+
+  def turns_chars_into_morse(word_array)
```

Comment:
This is soooo close!  This is giving you the array of letters that you want, now you just need to join those letters together!